src/transport/scheduler/xqc_scheduler_minrtt.c

```diff
@@ -103,24 +99,20 @@ xqc_minrtt_scheduler_get_path(void *scheduler,
                 best_path[path_class] ? best_path[path_class]->path_id : -1);
     }
 
-
-    for (path_class = XQC_PATH_CLASS_AVAILABLE_HIGH; 
-         path_class < XQC_PATH_CLASS_PERF_CLASS_SIZE; 
-         path_class++)
+    if (best_path_class != INVALID_PATH_CLASS
```

Comment:
In the original implementation. The scheduler classifies paths into different classes according to their bandwidth/rtt/loss_rate/etc. The classes have different priorities and a lower index leads to a higher priority. For each class, we need to find the path whose RTT is the minimum in the class. At the end, we will select the minrtt path from the highest priority class. So, your code breaks our original logic. 

Indeed, the name of the scheduler, "minrtt", may not exactly describe what it does. It's a bit confusing.

Reply:
ok, thanks for the explanation about this path's classes. I will revert original version of this piece of code